dangerfile.js

```diff
@@ -60,11 +60,16 @@ Sentry runs a symbolicator integration test suite located at [\`tests/symbolicat
 Follow these steps to update snapshots in Sentry:
 
 1. Check out latest Sentry \`master\` and enable the virtualenv.
-2. Stop the symbolicator devservice using \`sentry devservices down symbolicator\`.
-3. Run your development symbolicator on port \`3021\`.
-4. Export \`SENTRY_SNAPSHOTS_WRITEBACK=1\` and run symbolicator tests with pytest.
-5. Review snapshot changes locally, then create a PR to Sentry.
-6. Merge the Symbolicator PR, then merge the Sentry PR.
+2. Make sure your other devservices are running via \`sentry devservices up\`, and that symbolicator
+is enabled (\`symbolicator: true\`) in sentry via \`~/.sentry/config.yml\`.
```

Comment:
The config seems important enough (esp if it's not default) to warrant its own line item.

Reply:
good point, i've split this out 👍